memgpt/client/client.py

```diff
@@ -236,7 +236,22 @@ def list_agents(self):
         response = requests.get(f"{self.base_url}/api/agents", headers=self.headers)
         return ListAgentsResponse(**response.json())
 
+    def get_agent_id_from_name(self, agent_name: str) -> uuid:
+        agents = self.list_agents()
+        if agents.num_agents > 0:
```

Comment:
Changing order of expressions may improve readability.
```python
if not agents.num_agents:
  raise ValueError("no agents available")
agent_name = [agent["id"] for agent in agents.agents if agent["name"] == agent_name]
if not agent_name:
  raise NameError(f"no {agent_name} agent found")
return agent_name[0]
```
Same code, but success path was simplified to less nesting levels. What do you think?

Reply:
I appreciate your comment. With another fix, the code will be as follows:

``` python
def get_agent_id_from_name(self, agent_name: str) -> uuid:
  agents = self.list_agents()
  if not agents.num_agents:
      raise ValueError("no agents available")
  agent_id = next((agent["id"] for agent in agents.agents if agent["name"] == agent_name), None)
  if not agent_id:
      raise NameError(f"No agent named {agent_name} found")
  return agent_id  
```